packages/platforms/celestica/x86-64/x86-64-cel-redstone-xp/onlp/builds/src/module/src/sfp_xfp.h

```diff
@@ -0,0 +1,294 @@
+#ifndef INCLUDE_SFP_XFP
```

Comment:
What are all of these defines used for? All SFF defines should be available in the SFF module:
https://github.com/opencomputeproject/OpenNetworkLinux/tree/master/packages/base/any/onlp/src/sff/module/inc/sff

Any settings missing should probably be added there so everyone can use them.